src/Services/EncounterService.php

```diff
@@ -234,7 +252,17 @@ class_code,
         try {
             $whereFragment = FhirSearchWhereClauseBuilder::build($search, $isAndCondition);
             $sql .= $whereFragment->getFragment();
-            $sql .= " ORDER BY fe.eid DESC";
+
+            if (empty($options['order'])) {
+                $sql .= " ORDER BY fe.eid DESC";
+            } else {
+                $sql .= " ORDER BY " . $options['order'];
```

Comment:
where is $options coming from and what are some examples of what it will look like?
(just trying to figure out if it needs to be escaped)